pants

```diff
@@ -65,8 +98,14 @@ export PY="${PY:-python3}"
 # Pants will try to use Python 2 for subprocesses. Without setting minor version constraints, when using Python 3
 # we could end up using a different Python minor version for subprocesses than we do for Pants.
 py_major_minor_patch=$(${PY} -c 'import sys; print(".".join(map(str, sys.version_info[0:3])))')
+
```

Comment:
We don't want this extra whitespace because the comment explains both the above and the below line.

Reply:
Thanks!